Altis_Life.Altis/stringtable.xml

```diff
@@ -1562,6 +1562,30 @@
             <Polish>Admin Message Sent To All: %2</Polish>
             <Chinesesimp>管理员发送信息到全体: %2</Chinesesimp>
         </Key>
+        <Key ID="STR_CELLMSG_NoCops">
+            <Original>There are currently no police units available!</Original>
+            <Czech>There are currently no police units available!</Czech>
```

Comment:
I recommend using https://www.deepl.com/translator when possible, and google translate for languages not available on deepl so we have some basic translations for every language.

Reply:
good idea